src/app/page.tsx

```diff
@@ -15,8 +16,10 @@ export default function Home() {
           <div id="faqSection">
             <FAQ />
           </div>
-
-          <Venue />
+          <div id="venueSection">
```

Comment:
what is `id` for?

Reply:
so the id were used to scroll to that section
but the buttons are removed i guess they wont be needed